beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/path/aws/S3BytesDeletedCalculator.java

```diff
@@ -55,4 +61,10 @@ public long getBytesDeleted() {
     return bytesDeleted;
   }
 
+  private void checkKeysAreEmpty() {
+    if (!keyToSize.isEmpty()) {
+      throw new BeekeeperException("Should not store file sizes twice.");
```

Comment:
I don't see this exception caught anywhere, so is this an error that would make the whole process to stop?

Reply:
+1 this looks odd when does this happen, perhaps we should just log a warning instead of throwing an exception?

Reply:
Yep happy to log instead. Just wanted to enforce this class only being used once before being thrown away as in my head there's no "add keys" option, but actually that's exactly what the method does. So it's actually fine if this method is reused.

Reply:
I've removed the exception, I didn't put the log in either, seemed unnecessary. Happy to put it in though if you prefer

Reply:
looks ok